leaflet/__init__.py

```diff
@@ -182,7 +183,8 @@ def _normalize_plugins_config():
                     # absolute URL or a URL starting at root
                     pass
                 else:
-                    urls[i] = urljoin(settings.STATIC_URL, url)
```

Comment:
I'll add a new PR
@leplatrem I suggest you add instructions on how to locally run the same test as Travis -- python quicktest.py leaflet --db=postgres (which I had to figure out for myself) passed perfectly. Contributing without knowing the tests is no fun.